### PR TITLE
Feat/swap from v2

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: src/routes/TraderForm/Main/PoolHeader/index.tsx [hunk 1]
```diff
@@ -55,7 +56,13 @@ export const PoolHeader: React.FunctionComponent<PoolHeaderProps> = () => {
         <VariableBox>
           <LabelTokenTypography
             colorToken="lavenderWeb"
-            // differenceValue={-2} //TODO Alex
+            differenceValue={
+              variableRateInfo24hAgo.status === 'success' && variableRateInfo.status === 'success'
```
Comment (ssbarbee) [inline src/routes/TraderForm/Main/PoolHeader/index.tsx:60]: why not move this to selector? This entire logic can live into a selector that returns either a formatted number or undefined. 

Might be best place for it.

File: src/routes/TraderForm/Form/MarginAmountField/index.tsx [hunk 2]
```diff
@@ -19,6 +20,9 @@ export const MarginAmountField: React.FunctionComponent<NotionalAmountProps> = (
   const infoPostSwap = useAppSelector(selectInfoPostSwap);
   const aMM = useAppSelector(selectSwapFormAMM);
 
+  const marginRequiredError = marginAmount.error !== null && marginAmount.error !== 'WLT';
```
Comment (ssbarbee) [inline src/routes/TraderForm/Form/MarginAmountField/index.tsx:23]: perhaps move this to selector as well. That returns string or undefined then no need for the cast `as string` below.

File: src/routes/TraderForm/Form/MarginAmountField/index.tsx [hunk 3]
```diff
@@ -19,6 +20,9 @@ export const MarginAmountField: React.FunctionComponent<NotionalAmountProps> = (
...
+  const walletError = marginAmount.error === 'WLT';
```
Comment (ssbarbee) [inline src/routes/TraderForm/Form/MarginAmountField/index.tsx:24]: perhaps move this to selector as well.

File: src/routes/TraderForm/Form/MarginAmountField/index.tsx (hunk 2 above)
Comment (avniculae) [reply src/routes/TraderForm/Form/MarginAmountField/index.tsx:23]: This is a boolean, we would still need casting

File: src/routes/TraderForm/Form/MarginAmountField/index.tsx (hunk 3 above)
Comment (avniculae) [reply src/routes/TraderForm/Form/MarginAmountField/index.tsx]: [753f5ee](https://github.com/Voltz-Protocol/voltz-ui/pull/576/commits/753f5ee4ee927fec2af52222bba233742d903d88)

File: src/routes/TraderForm/Form/MarginAmountField/index.tsx (hunk 2 above)
Comment (avniculae) [reply src/routes/TraderForm/Form/MarginAmountField/index.tsx]: [753f5ee](https://github.com/Voltz-Protocol/voltz-ui/pull/576/commits/753f5ee4ee927fec2af52222bba233742d903d88)

File: src/routes/TraderForm/Main/PoolHeader/index.tsx (hunk 1 above)
Comment (avniculae) [reply src/routes/TraderForm/Main/PoolHeader/index.tsx]: [753f5ee](https://github.com/Voltz-Protocol/voltz-ui/pull/576/commits/753f5ee4ee927fec2af52222bba233742d903d88)

**Review by ssbarbee — APPROVED** [review]

**Pull request merged**